Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+beautifulsoup4 = "~=4.12"
+selenium = "~=4.17"
+tenacity = "~=8.2"
+fire = "~=0.5"
+jsonlines = "~=4.0"
+pandas = "~=2.0"
+requests = "~=2.31"
+chardet = "~=5.2"
+html5lib = "~=1.1"
+xmltodict = "~=0.13"
+
+[dev-packages]
+black = "24.2.0"
```

Comment:
It is generally recommended to pin exact versions of any code formatter since even micro version changes can be breaking.